packages/flutter/lib/src/services/semantics_service.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'system_channels.dart';
+
+/// Allows access to the semantic feedback interface on the device.
```

Comment:
This should talk in more detail about what the "semantic feedback interface" is. Someone who doesn't know what that is wouldn't learn anything from this documentation.

Reply:
See if you like it better now.